mindpype/kernels/running_average.py

```diff
@@ -14,28 +14,37 @@
 
 class RunningAverageKernel(Kernel):
     """
-    Kernel to calculate running average across multiple trials in a session. Trials are automatically included into the next running average
+    Kernel to calculate running average across multiple trials in a session. 
+    Trials are automatically included into the next running average
     calculation.
 
+    .. note::
```

Comment:
Let's limit these notes to kernels to where the external function we are referencing is the primary function of the kernel. For example, for the mean kernel, we would make reference to the numpy.mean function because that's how we implemented it and the parameters used by the kernel will mirror the parameters used by numpy.mean. For this kernel, we are using the numpy functions as part of something larger, so we don't need to make explicit reference to all the numpy functions we use.